sdk/monitor/Azure.Monitor.Query/tests/QueryTimeSpanTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using NUnit.Framework;
+
+namespace Azure.Monitor.Query.Tests
+{
+    public class QueryTimeSpanTests
+    {
+        [Test]
+        public void ToStringDurationTests()
+        {
+            Assert.AreEqual("PT23M", new QueryTimeSpan(TimeSpan.FromMinutes(23)).ToString());
+        }
+
+        [Test]
+        public void ToStringDurationEndTests()
+        {
+            Assert.AreEqual("PT23M/2021-05-04T03:02:01.0000000Z", new QueryTimeSpan(TimeSpan.FromMinutes(23), new DateTimeOffset(2021, 5, 4, 3, 2, 1, TimeSpan.Zero)).ToString());
```

Comment:
```suggestion
            var myDt = new DateTimeOffset(2021, 5, 4, 3, 2, 1, TimeSpan.Zero);
            int durationMinutes = 23;

            Assert.AreEqual("PT{durationMinutes}M/{myDt.ToString("O")}", new QueryTimeSpan(TimeSpan.FromMinutes(durationMinutes), myDt).ToString());
```

Reply:
Why do you hate long unreadable asserts?